src/assets/text/monitoringText.js

```diff
@@ -1,24 +1,25 @@
 export default {
    textContents: {
-        title1: "",//"We know a lot about temperature. But there's a lot we don't know too.",
-        paragraph1: "At first glance, it appears that there is a lot of information about stream temperature in the Basin. The map below shows all of the locations where the USGS or other agencies have ever measured the temperature of medium to large streams.",
-        paragraph3: "The USGS has been monitoring temperature in the Delaware River Basin since 1901, and has amassed over 450,000 site-days worth of temperature measurements. Temperature monitoring really picked up in the mid-1960s, and has grown considerably in recent decades.",
-        paragraph4: "Since 2017, an uptick in temperature monitoring can be attributed to the USGS's Next Generation Water Observing Systems (NGWOS). NGWOS is a targeted effort in the Delaware River Basin to develop and apply new monitoring techniques for a wide range of water characteristics.",
-        paragraph5: "Temperature is one of the most widely measured water characteristics in the U.S. – temperature can now be monitored by boat, by camera, by drone, by cable, by satellite, by sensor, or by the tried and true method of a field technician sticking a thermometer into the water.",
+        title1: "The Delaware River Basin",//"We know a lot about temperature. But there's a lot we don't know too.",
+        paragraph1: "The Delaware River Basin covers 13,500 square miles in parts of four states, including New York, New Jersey, Pennsylvania, and Delaware. The Delaware River is rich in history, ecologically diverse, and critical to the regional economy.",
+        paragraph2: "Water managers in this region have a long history of applying innovative, regional solutions to ensure the long-term sustainability of this resource, which provides drinking water to over 15 million people in the region.",
```

Comment:
"region" is in this sentence 3 times, I would keep just the first

Reply:
Good catch, thanks